test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -1041,7 +1063,13 @@ class test_ucp_sockaddr_protocols : public test_ucp_sockaddr {
             << "recv_buf: '" << ucs::compact_string(recv_buf, 20) << "'";
     }
 
-    void test_tag_send_recv(size_t size, bool is_exp, bool is_sync = false)
+    typedef void (*stop_cb_t)(void *arg);
+
+    void test_tag_send_recv(size_t size, bool is_exp,
+                            bool is_sync = false,
+                            stop_cb_t send_stop = NULL,
+                            stop_cb_t recv_stop = NULL,
```

Comment:
looks like they are mutually exclusive, can use just one var for simplicity

Reply:
added the tests for simultaneous send/recv stop